### ISSUE TITLE
Unit test failure in DQM/Integration

### ISSUE THREAD

**Issue by silviodonato** [cms-sw/cmssw#31592]
We are getting 
```
----- Begin Fatal Exception 26-Sep-2020 16:17:29 CEST-----------------------
An exception of category 'PluginNotFound' occurred while
   [0] Constructing the EventProcessor
Exception Message:
Unable to find plugin 'L1GtTriggerMaskTechTrigTrivialProducer' in category 'CMS EDM Framework ParameterSet Description'. Please check spelling of name.
----- End Fatal Exception -------------------------------------------------
```
in the test `TestDQMOnlineClient-scal_dqm_sourceclient` of `DQM/Integration`.
I suspect it is related to https://github.com/cms-sw/cmssw/pull/31548/ (@guitargeek)

**Comment by silviodonato** [comment]
assign dqm

**Comment by guitargeek** [comment]
No idea how this issue relates to my cleaning PR, but I can't deny it looks like there is a connection because the issue was not present before my PR got merged. Hence, I jumped in to investigate and found a fix.

**Comment by guitargeek** [comment]
Sorry, actually the connection is obvious. The relevant package that was mistakenly a library instead of a plugin was previously included in a different plugin: https://github.com/cms-sw/cmssw/pull/31548/files#diff-d35b5dbd3e30c40bee4cc0c5bd6ae022L13.

That's how the plugin ended up in the register before I suppose.

**Comment by jfernan2** [comment]
+1

**Comment by silviodonato** [comment]
fixed by #31593
